Client/Frontend/Library/Bookmarks/BookmarksPanelViewModel.swift

```diff
@@ -60,10 +68,73 @@ class BookmarksPanelViewModel {
     func didAddBookmarkNode() {
         flashLastRowOnNextReload = true
     }
-
+    
+    func bookmarkExportSelected(in viewController: BookmarksPanel, onDone: @escaping (Error?) -> Void) {
+        Task {
+            self.onExportDoneHandler = onDone
+            do {
+                let bookmarks = try await getBookmarksForExport()
+                try await bookmarksExchange.export(bookmarks: bookmarks, in: viewController, barButtonItem: viewController.moreButton)
+                await notifyExportDone(nil)
```

Comment:
Looks like `notifyExportDone` executes the received `onDone` on main, I guess the method was split to use `@MainActor`, but how about using it directly this to reduce code:
```suggestion
            Task { @MainActor in
                onDone(nil)
            }
```
I believe this would reach the same behaviour, but let me know if I'm missing something 🙏
(This also applies for the usage inside the `catch` below)

Reply:
Yes it does indeed, I actually thought my version is less verbose because it has less duplication, but effectively you might be right. Let me check...

Reply:
Hm, I don't know, tbh. I would prefer:

```swift

                await notifyExportDone(nil)
            } catch {
                await notifyExportDone(error)
            }
```

over

```swift
                Task { @MainActor in onExportDoneHandler?(nil) }
            } catch {
                Task { @MainActor in onExportDoneHandler?(error) }
            }
```

even if there is one method more, just for the sake of having one less closure, but maybe @d4r1091 has another input / opinion about this?

Reply:
Your point is also valid, feel free to leave as you prefer as well 😁

I think I would still find the below cleaner since it gets rid of the method and the intermediate closure variable, but this is also personal opinion since we don't have a defined style, so feel free to leave it like it is 🙂
```
                Task { @MainActor in onDone(nil) }
            } catch {
                Task { @MainActor in onDone(error) }
            }
```

Also noticed the `handlePickedUrl` method further down this file has a similar style, so if you'd like you can try and match both to keep the same in this file.

Reply:
Ha! You got me with this one 😄  yes I agree that #1 priority should be to be consistent here, I'm fine with both solutions, I'm leaning towards separate method it `@MainActor` annotation just for the sake of removing one closure, but I'M also fine with your proposed solution.

Reply:
No worries, you can keep the one you are leaning towards then, both are good alternatives indeed 😁